wagtail/admin/wagtail_hooks.py

```diff
@@ -998,6 +998,23 @@ def register_editors_guide_menu_item():
     )
 
 
+@hooks.register("register_help_menu_item")
+def register_keyboard_shortcuts_menu_item():
```

Comment:
Let's add a simple docstring here, helps for future context. I can add this when merging.

```suggestion
def register_keyboard_shortcuts_menu_item():
    """
    Triggers the keyboard shortcuts dialog to open when clicked
    while preventing the default link click action.
    """

```